### ISSUE TITLE
Flatpak build

### ISSUE THREAD

**Issue by LeandroStanger** [Fmstrat/winapps#113]
Please, consider supporting [Flatpak](http://docs.flatpak.org/en/latest/introduction.html#reasons-to-use-flatpak) and publishing WinApps on [Flathub](https://github.com/flathub/flathub/wiki/App-Submission).

**Comment by Calinou** [comment]
Since Winapps needs to run a virtual machine using KVM, is this even something that can be sandboxed in a Flatpak?

**Comment by fedelibre** [comment]
Yes, I think so. For example, GNOME Boxes [bundles QEMU](https://github.com/flathub/org.gnome.Boxes/blob/master/org.gnome.Boxes.json#L324).

**Comment by aaylnx** [comment]
A flatpak would be great.

**Comment by kojid0** [comment]
Winapps would get more attention if it was on flathub, hopefully we'll see it there soon
